mobile/src/navigations/root/root.tsx

```diff
@@ -28,7 +28,7 @@ type Properties = {
 
 const Root: React.FC<Properties> = ({
     isSignedIn = false,
-    isProfileComplete = true,
+    isProfileComplete = false,
     role = 'talent',
 }) => {
     // prettier-ignore
```

Comment:
why do we ignore prettier?

Reply:
Because we have a conflict between linter and prettier, we've decided temporarily to place `prettier-ignore` there

Reply:
There are multiple `? :` checks and prettier removes brackets and there is linter error

Reply:
You can move this logic to renderStackScreen function for example and use early return to prevent multiple ? :

Reply:
And in this case it would be more human readable as for me

Reply:
@katerynakostikova what do you think about this idea ?

Reply:
Let's do like this

Reply:
yes

Reply:
Done